src/main/webapp/style.css

```diff
@@ -0,0 +1,47 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  size:80%;
+  
+}
+h1 {
+  background-color: #00a6a6;
```

Comment:
For accessibility (specifically here, those with poor eye sight), it's good to have strong contrast between the foreground (the text) and the background. With white as the text color and #00a6a6 as the background, it's considered to have low contrast for people with bad eye sight. You can check it by using contrast checkers like this one:

https://webaim.org/resources/contrastchecker/

A similar color that has a bit more contrast is #006161, but I recommend playing with the tool a bit and seeing something you like that has more contrast.